src/media/js/main.js

```diff
@@ -49,7 +49,8 @@ require(
 
     z.page.on('loaded logged_in', function() {
         if (user.logged_in() && !user.get_permission('curator') &&
-            !user.get_permission('admin')) {
+            !user.get_permission('admin') &&
+            utils.baseurl(location.pathname) !== urls.reverse('403')) {
```

Comment:
This is unrelated but prevents the site from redirecting to `/403` if you're already on the `/403` page which locks up the browser.